AnnoDesigner/Helper/PolygonBoundaryFinderHelper.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+namespace AnnoDesigner.Helper
+{
+    public static class PolygonBoundaryFinderHelper
+    {
+        private enum Direction
+        {
+            Up,
+            Left,
+            Down,
+            Right
+        }
+
+        private static (int x, int y) GetLeftPixel((int x, int y) point, Direction direction)
```

Comment:
This method should return Point, rather than a ValueTuple. It saves on unboxing calls as the rest of the codebase uses Point.

nit: Also, I am not sure about the word Pixel in the method name - the places where this could be used could mean we are dealing with pixels, grid units, etc.  - not sure what we'd change it to though - just something for your consideraton.

I can see why we may want to use an int-based point, rather than a double-based one - could be worth experimenting with in future.

Reply:
Renamed to GetLeftCell